apps/src/templates/rubrics/ProgressRing.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import i18n from '@cdo/locale';
+import style from './rubrics.module.scss';
+import {learningGoalShape} from './rubricShapes';
+
+export default function ProgressRing({
+  learningGoals,
+  currentLearningGoal,
+  understandingLevels,
+  radius,
+  stroke,
+}) {
+  const normalizedRadius = radius - stroke * 2;
+  const circumference = normalizedRadius * 2 * Math.PI;
+  const assessedLearningGoals = understandingLevels.filter(u => u >= 0).length;
+  const currentLearningGoalOffset =
+    circumference -
+    ((currentLearningGoal + 1) / learningGoals.length) * circumference;
+  const assessedLearningGoalOffset =
+    circumference -
+    (assessedLearningGoals / learningGoals.length) * circumference;
+
+  return (
+    <svg height={radius * 2} width={radius * 2}>
+      <circle
+        className={style.progressRing}
+        stroke="#D4D5D7"
+        fill="transparent"
+        strokeWidth={stroke}
+        r={normalizedRadius}
+        cx={radius}
+        cy={radius}
+      />
+      <circle
+        className={style.progressRing}
+        stroke="#A9ACAF"
+        fill="transparent"
+        strokeWidth={stroke}
+        strokeDasharray={circumference + ' '}
+        style={{strokeDashoffset: currentLearningGoalOffset}}
+        r={normalizedRadius}
+        cx={radius}
+        cy={radius}
+      />
+      <circle
+        className={style.progressRing}
+        stroke="#0093A4"
+        fill="transparent"
+        strokeWidth={stroke}
+        strokeDasharray={circumference + ' '}
+        style={{strokeDashoffset: assessedLearningGoalOffset}}
+        r={normalizedRadius}
+        cx={radius}
+        cy={radius}
+      />
+      <text className={style.progressRingText} x="50%" y="50%" dy="0.3em">
+        {currentLearningGoal + 1} {i18n.of()} {learningGoals.length}
```

Comment:
This looks great - I think we typically have fewer than 10 learning goals, but out of curiousity, how does this look/work if it is reading `10 of 10` - it seems like it wouldn't fit in the circle... 

Again, I am reasonably sure we have less than 10 items on every rubric, but perhaps something to check with Molly and the Curriculum Team on.

Reply:
Good point. That's probably something we need to talk to Molly and designer Mark about. I think you're right that 2 digit values wouldn't fit.